src/Features/CSharp/Portable/Completion/CompletionProviders/SymbolCompletionProvider.cs

```diff
@@ -27,12 +28,52 @@ protected override Task<ImmutableArray<ISymbol>> GetSymbolsAsync(SyntaxContext c
                 context.SemanticModel, position, context.Workspace, options, cancellationToken);
         }
 
+        internal async override Task<bool> ShouldProvidePreselectedItemsAsync(CompletionContext completionContext, SyntaxContext syntaxContext, Document document, int position, Lazy<ImmutableArray<ITypeSymbol>> inferredTypes, OptionSet options)
+        {
+            var sourceText = document.GetTextSynchronously(CancellationToken.None);
+            if (ShouldTriggerInArgumentLists(sourceText, options))
+            {
+                if (completionContext.Trigger.Kind == CompletionTriggerKind.Insertion &&
+                    await IsTriggerInArgumentListAsync(document, position - 1, CancellationToken.None).ConfigureAwait(false) == true)
+                {
+                    return false;
```

Comment:
when actually submitting this provide an explanation here.  you explained well over Teams.  We just want this in the code to make sense.

Reply:
Documented.